src/components/message/message-card/MessageHeader.tsx

```diff
@@ -19,15 +19,16 @@ import { EventMessage, ParsedMessage } from '../../../models/EventMessage';
 import { createStyleSelector } from '../../../helpers/styleCreators';
 import { formatTime, timestampToNumber } from '../../../helpers/date';
 import { getHashCode } from '../../../helpers/stringHash';
+import MessageCardTools, { MessageCardToolsProps } from './MessageCardTools';
 
 interface MessageInfoProps {
 	message: EventMessage;
-	parsedMessage: ParsedMessage | null;
+	parsedMessage?: ParsedMessage;
```

Comment:
we use  MessageHeader only inside MessageCardBase component, and parsedMessages is always defined, why do we need this message/parsedMessage implicit rendering